compiler/Ballerina.toml

```diff
@@ -2,3 +2,4 @@
 org = "wso2"
 name = "nballerina"
 version = "0.1.0"
+export = ["nballerina.types.regex", "nballerina.types"]
```

Comment:
Why do we need to export `nballerina.types` given that `extra/regex` uses only `nballerina.types.regex`?

Reply:
fixed